libs/server/Auth/GarnetACLAuthenticator.cs

```diff
@@ -25,15 +25,19 @@ class GarnetACLAuthenticator : IGarnetAuthenticator
         /// </summary>
         User _user = null;
 
+
+        private IGarnetAuthenticator _authenticator;
+
         /// <summary>
         /// Initializes a new ACLAuthenticator instance.
         /// </summary>
         /// <param name="accessControlList">Access control list to authenticate against</param>
         /// <param name="logger">The logger to use</param>
-        public GarnetACLAuthenticator(AccessControlList accessControlList, ILogger logger)
+        public GarnetACLAuthenticator(AccessControlList accessControlList, ILogger logger, IGarnetAuthenticator wrapperAuthenticator = null)
```

Comment:
Please always add ILogger as the last parameter. Also, don't forget to update the summary for the new parameter.
```suggestion
public GarnetACLAuthenticator(AccessControlList accessControlList, IGarnetAuthenticator wrapperAuthenticator = null, ILogger logger = null)
```